internal/artifacts/meta_test.go

```diff
@@ -190,6 +191,48 @@ cli_version:["x"]
 			Ω(mtarPath).Should(BeAnExistingFile())
 		})
 	})
+	Describe("ExecuteMerge", func() {
+		It("Succeeds with single mtaext file", func() {
+			resultFileName := "result.yaml"
```

Comment:
I should move resultFileName out of It's.

Reply:
Done